lib/routes/user/home/invoice_bottom_sheet.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:barcode_scan/barcode_scan.dart';
+import 'package:breez/bloc/invoice/invoice_bloc.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:flutter/material.dart';
+
+class InvoiceBottomSheet extends StatefulWidget {
+  final InvoiceBloc invoiceBloc;
+  final bool isSmallView;
+
+  InvoiceBottomSheet(this.invoiceBloc, this.isSmallView);
+
+  @override
+  State createState() => InvoiceBottomSheetState();
+}
+
+class InvoiceBottomSheetState extends State<InvoiceBottomSheet> with TickerProviderStateMixin {
+  bool isExpanded;
+
+  @override
+  void initState() {
+    super.initState();
+    isExpanded = false;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedContainer(
+        transform: isExpanded ? Matrix4.translationValues(0, 0, 0) : Matrix4.translationValues(0, 112.0, 0),
+        duration: Duration(milliseconds: 150),
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.end,
+            crossAxisAlignment: CrossAxisAlignment.end,
+            mainAxisSize: MainAxisSize.max,
+            children: <Widget>[
+              _buildInvoiceMenuItem(widget.isSmallView, "INVOICE", "src/icon/invoice.png", () {
+                setState(() {
+                  isExpanded = !isExpanded;
+                });
+              }, isFirst: true),
+              _buildInvoiceMenuItem(widget.isSmallView, "PAY", "src/icon/qr_scan.png", () async {
+                String decodedQr = await BarcodeScanner.scan();
+                widget.invoiceBloc.decodeInvoiceSink.add(decodedQr);
+              }),
+              _buildInvoiceMenuItem(
+                  widget.isSmallView, "CREATE", "src/icon/paste.png", () => Navigator.of(context).pushNamed('/create_invoice')),
+            ]));
+  }
+
+  Widget _buildInvoiceMenuItem(bool isSmallView, String title, String iconPath, Function function, {bool isFirst = false}) {
```

Comment:
`isSmallView` is a class variable which this function has access to. No need to pass it as argument.